wagtail/admin/templates/wagtailadmin/shared/header.html

```diff
@@ -27,7 +28,7 @@ <h1 class="w-header__title" id="header-title">
                     {% if icon %}
                         {% icon class_name="w-header__glyph" name=icon %}
                     {% elif avatar %}
-                        <div class="w-header__glyph avatar"><img src="{{ avatar }}" alt="" /></div>
+                        <div class="w-header__glyph avatar {{ avatar_size }}"><img src="{{ avatar }}" alt="" /></div>
```

Comment:
This is the only header with an avatar (not an icon) - so you probably can avoid yet another variable and just have this always showing as medium. 

Or better yet - avoid adding a medium avatar variation completely.

Reply:
Good point! I've revised this now